tissot.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Sri Krishna (neonmicky)",
+			"oem": "Xiaomi",
+			"device": "Xiaomi Mi A1",
+			"filename": "crDroidAndroid-12.0-20220120-tissot-v8.1.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/tissot/8.x/crDroidAndroid-12.0-20220120-tissot-v8.1.zip/download",
+			"timestamp": 1642623167,
+			"md5": "c887501c8144d6ff151d7e24f464dd81",
+			"sha256": "04405c8d3aa255f68316bc41e2f910b919a99e8e3b985e5beb17676446544d7b",
+			"size": 715603226,
+			"version": "8.1",
+			"buildtype": "Monthly",
```

Comment:
Release as BETA of fix
encryption on by default as per new rules
https://github.com/crdroidandroid/android_device_xiaomi_msm8953-common-tissot/blob/11.0/rootdir/fstab.qcom#L11#L12